### DISCUSSION TITLE
Ask questions, share bugs, and chat using discussions now!

### DISCUSSION THREAD

**Discussion by LunarTaku** [LunarTaku/djs-welcome-system#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
 * Share bugs and propose fixes.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devshina** [comment]
Hi i dont know how to put mongo db connection can you help me

https://sourceb.in/7yhjqFni6M
someone fixed my ready.js ? please. with mongoose
